docs/Guides/GitOnZOS.md

```diff
@@ -0,0 +1,59 @@
+# Git on z/OS
+
+Git on z/OS is a version of Git that has been adapted to work on the IBM mainframe operating system, z/OS. It allows developers to take advantage of Git's powerful version control capabilities while addressing the unique challenges of the mainframe environment. With Git on z/OS, developers can manage their code, collaborate with other developers, and maintain a clear history of the changes that have been made to the codebase.
+
+You can find more information and download the software on the Git on z/OS GitHub page: https://github.com/ZOSOpenTools/gitport
+
+## Encoding considerations
+
+One of the key features of Git on z/OS is its support for various encodings. This is achieved through the use of the `.gitattributes` file, which can be specified globally or locally in repositories. The `.gitattributes` file determines the encoding of the working tree files and can be used to convert them to a specific encoding, such as IBM-1047. 
+
+For example, if you want to convert all files in your repository from Git's internal UTF-8 encoding to IBM-1047, you can add the following line to your `.gitattributes` file:
```

Comment:
Do you want to mention that IBM-1047 is an EBCDIC encoding commonly used in North America?